src/components/__stories__/modal.tsx

```diff
@@ -0,0 +1,45 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+
+import Button from "../buttons/default"
+import Modal from "../modal/modal"
+import ModalHeader from "../modal_header"
+import Title from "../title"
+
+class ModalDemo extends React.Component<any, any> {
```

Comment:
This is an example component, since the generic modal sort of only makes sense by being launched/triggered from some parent container.